GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1255,7 +1255,11 @@ private void OnGridViewKeyUp(object sender, KeyEventArgs e)
 
                 case Keys.Delete:
                     {
-                        InitiateRefAction(
+                        // Check modifier detail if pressed.
+
+                        if (e.Modifiers == Keys.None)
```

Comment:
Please invert `if` to reduce nesting.

Reply:
If i add the "if" clause before switch then i guess nesting is reduced.